files/image_config/monit/container_checker

```diff
@@ -88,14 +98,12 @@ def check_dockers(always_running_containers):
               in running state.
```

Comment:
Comment says returns containers that are *not* in running state, but code was updated to return running containers

Reply:
Thanks. Corrected.